### DISCUSSION TITLE
Project Init

### DISCUSSION THREAD

**Discussion by thepushkarp** [iiitv/alumni-portal#1] [General]
Hi @iiitv/alumni-portal-team  🎊!

Great to have you on this project. You can start working on the project here and make relevant issues and open a pull request for the same, Feel free to ping @iiitv/alumni-portal-mentors if you need any help.

To keep a track of current progress, we can use the kanban board under 'Projects' (https://github.com/iiitv/alumni-portal/projects/1) for tracking 'To do', 'In Progress' and 'Completed Tasks' and the issues/PRs linked to them.

You can start with adding the initial set of tasks there and start working on the same :v:.

Let us know here if there's anything we can help with.

**Comment by thepushkarp** [comment]
@iiitv/alumni-portal-team any updates on the progress?

**Reply by solankidivyam** [reply]
We have started with UI of the landing page and other pages as scheduled in the Proposal. We will start with the frontend in coming few days, the implementation of the landing page is scheduled in this weeks agenda😁.
